### ISSUE TITLE
[Bug]Can't use iGPU/VPU on OpenVINO open source build on Ubuntu16.04

### ISSUE THREAD

**Issue by cosmos1982** [openvinotoolkit/openvino#6394]
### Platform:
Ubuntu16.04, python3.7.4, cmake 3.18.4
### Description:
OpenVINO source code can successfully built on Ubunt16.04 with configuration -DENABLE_VPU=OFF -DENABLE_CLDNN=OFF. But if I want to enable GPU or VPU,  errors will occur:
 1. OpenCL drivers 19.41.14441 required on [wiki](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux) can't be installed on Ubuntu16.04 which has an error:
 ```
Setting up intel-gmmlib (19.3.2) ...
dpkg: dependency problems prevent configuration of intel-igc-core:
 intel-igc-core depends on libc6 (>= 2.27); however:
  Version of libc6:amd64 on system is 2.23-0ubuntu11.3.

dpkg: error processing package intel-igc-core (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of intel-igc-opencl:
 intel-igc-opencl depends on intel-igc-core (= 1.0.2597); however:
  Package intel-igc-core is not configured yet.

dpkg: error processing package intel-igc-opencl (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of intel-ocloc:
 intel-ocloc depends on intel-igc-opencl (= 1.0.2597); however:
  Package intel-igc-opencl is not configured yet.

dpkg: error processing package intel-ocloc (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of intel-opencl:
 intel-opencl depends on intel-igc-opencl (= 1.0.2597); however:
  Package intel-igc-opencl is not configured yet.
 intel-opencl depends on libstdc++6 (>= 6); however:
  Version of libstdc++6:amd64 on system is 5.4.0-6ubuntu1~16.04.12.

dpkg: error processing package intel-opencl (--install):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.23-0ubuntu11.2) ...
Errors were encountered while processing:
 intel-igc-core
 intel-igc-opencl
 intel-ocloc
 intel-opencl
```
I also have tried the opencl driver [18.45.11804](https://github.com/intel/compute-runtime/releases/tag/18.45.11804) which is expected to be compatible on Ubuntu16.04. This driver can be installed, and open source can also be built without disabling CLDNN, but when I run 'hello_query_devices' sample,  no GPU device is found.

2. Without -DENABLE_VPU=OFF, I got this error:
```
CMake Error at inference-engine/thirdparty/movidius/XLink/XLink.cmake:36 (message):
  libusb is required
Call Stack (most recent call first):
  inference-engine/thirdparty/movidius/XLink/CMakeLists.txt:7 (include)
```

How can I support iGPU and VPU? Is there any workaround? Thanks!

**Comment by vladimir-paramuzov** [comment]
@cosmos1982 GPU plugin can be built w/o any driver installed as OCL headers and ICD library are taken from our source tree, not from the system. 

As for the driver issues, 19.41.14441 driver from github was built on ubuntu 18, and can't be installed on ubuntu 16 with default C runtime. That's why you get `intel-igc-core depends on libc6 (>= 2.27); however: Version of libc6:amd64 on system is 2.23-0ubuntu11.3.` error

18.45 seems to be compatible, but maybe you have some other issue with it. Please check the following page for troubleshooting hints: https://github.com/openvinotoolkit/openvino/wiki/GPUPluginDriverTroubleshooting

**Comment by cosmos1982** [comment]
@vladimir-paramuzov Thanks for your quick reply!! I don't quite understand, did you mean I should recompile GPU plugin with opencl driver 18.45?

**Comment by vladimir-paramuzov** [comment]
@cosmos1982 As I said, GPU plugin compilation doesn't depend on any driver, i.e. you may have no installed driver at all, but you'll be still able to compile GPU plugin.

So please enable GPU plugin build via `-DENABLE_CLDNN=ON` cmake option and then try to install and setup 18.45 driver.

**Comment by cosmos1982** [comment]
@vladimir-paramuzov , got it, very clear, I'll try and give my feedback ASAP. Thanks!

**Comment by Iffa-Meah** [comment]
Hi @cosmos1982 ,

Do you require any further helps? if not we shall proceed in closing this thread.

**Comment by Iffa-Meah** [comment]
Closing issue, feel free to re-open or start a new issue if additional assistance is needed.
